smoke-test/pom.xml

```diff
@@ -119,8 +119,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.8.1</version>
         <configuration>
-          <source>11</source>
-          <target>11</target>
+          <source>10</source>
```

Comment:
Was there any need to change this ?

Reply:
not really, was changed by IDEA for some reasons.

Reply:
reverting back